BookReader/BookReader.js

```diff
@@ -3873,16 +3891,50 @@ BookReader.prototype.navigationIsVisible = function() {
     return tooltop == 0;
 };
 
+/**
+ * Main controller that toggles navigation.
+ * Defaults to SHOW the navigation chrome
+ */
+BookReader.prototype.toggleNavigation = function brToggleNavigation(hide) {
```

Comment:
Since we have the two public methods `hideNavigation` and `showNavigation`, does this method need to be public?

Also, this method name is kind of confusing when it's called with the intention of showing the menu. ie `br.toggleNavigation()` makes it sound like it's going to either show or hide the navigation (to the opposite of its current state) whereas that is going to cause the nav to show.

Reply:
Seems like all core functionality is currently bound to the `BookReader` prototype object.  I'd like to keep to the convention set. Also, since it is based on its instance, it makes sense for it to be part of the prototypal core, so we can call `this.toggleNavigation`

I do feel it can be better named. Will update name.